meson.build

```diff
@@ -60,6 +73,12 @@ nanoarrow_dep = declare_dependency(include_directories: [incdir],
 if get_option('ipc')
     flatcc_dep = dependency('flatcc')
 
+    install_headers(
+        'src/nanoarrow/nanoarrow_ipc.h',
+        # 'src/nanoarrnow/nanoarrow_ipc_flatcc_generated.h',
```

Comment:
I don't know a ton about flatbuffers but not sure if this file is required? I didn't see any internal usage

If we wanted it probably better to add a custom target to meson that runs the `./dev/update_fbs.sh` script, although Meson most likely will not want those updates to be done in the source tree

Reply:
Ah OK - yea it looks like the CMake install fails

```sh
CMake Error at cmake_install.cmake:78 (file):
  file INSTALL cannot find
  "/home/willayd/clones/arrow-nanoarrow/src/nanoarrow/nanoarrow_ipc_flatcc_generated.h":
  No such file or directory.
```